app/assets/javascripts/application.js

```diff
@@ -27,6 +27,7 @@
 //= require ./discourse/lib/formatter
 //= require ./discourse/lib/eyeline
 //= require ./discourse/lib/show-modal
+//= require ./discourse/lib/safari-hacks
```

Comment:
I would extract `isAppleDevice` from `safari-hacks`

Reply:
Yeah, I was thinking about moving it into `discourse/lib/utilities`. Not sure if there's even a better place.

Reply:
seems fine to me